server/proxy/grpcproxy/watch.go

```diff
@@ -238,7 +239,7 @@ func (wps *watchProxyStream) recvLoop() error {
 			if err := wps.checkPermissionForWatch(cr.Key, cr.RangeEnd); err != nil {
 				wps.watchCh <- &pb.WatchResponse{
 					Header:       &pb.ResponseHeader{},
-					WatchId:      -1,
+					WatchId:      int64(mvcc.InvalidWatchID),
```

Comment:
It seems you are still using `mvcc.InvalidWatchID `instead of `clientv3.InvalidWatchID`

Reply:
Sorry, I’m missing some place and I did a whole check on the commit.